src/main/resources/config.yml

```diff
@@ -1,38 +1,23 @@
-tnt:
-  log: true
-  disable-when-target-not-found: true
-creeper:
-  log: true
-  disable-when-target-not-found: true
-endercrystal:
-  log: true
-  disable-when-target-not-found: true
-fireball:
-  log: true
-  disable-when-target-not-found: true
-tntminecart:
-  log: true
-  disable-when-target-not-found: true
-bed:
-  log: true
-  disable-when-target-not-found: true
-respawn-anchor:
-  log: true
-  disable-when-target-not-found: true
-other-explosive-blocks:
-  disable-when-target-not-found: true
+block-explosion:
+  # Should we log and tracing for explosion?
+  enable: true
+  # Disable the explosion if no source can be tracking.
+  disable-unknown: true
+  # The message will broadcast to players that around 15 blocks nearby
+  alert: "&cAn explosion near you has been blocked because that caused by unknown object."
```

Comment:
&cAn explosion near you has been blocked because it was caused by an unknown object.